README.md

```diff
@@ -65,6 +65,22 @@ http://127.0.0.1:8000/kpm
 ```
 
 # Endpoint map
+- kpm: provides data from City of Portland Budget in Brief documents (e.g. FY 2016-17), all Service Area sections, table "Key Performance Measures".
 - ocrb: provides data from City of Portland Budget in Brief documents (e.g. FY 2016-17), all Service Area sections, table "Operating and Capital Requirements by Bureau".
 - summary: uses query parameters to return subsets of the budget data given by the 'ocrb' endpoint.
-- kpm: provides data from City of Portland Budget in Brief documents (e.g. FY 2016-17), all Service Area sections, table "Key Performance Measures".
+
+# Sample Endpoint data
+- KPM:
+```json
+[
+    {"source_document":"FY 2015-16 Budget in Brief","service_area":"Public Safety","bureau":"Bureau of Emergency Communications","key_performance_measures":"BOEC - % of priority medical calls dispatched within 90 seconds","fy":"2013-14","budget_type":"Actual ","amount":72.0,"units":"%"},{"source_document":"FY 2015-16 Budget in Brief","service_area":"Public Safety","bureau":"Bureau of Fire & Police Disability & Retirement","key_performance_measures":"FPDR - Tax levy rate per $1,000 of Real Market Value","fy":"2013-14","budget_type":"Actual ","amount":1.62,"units":"$"},    {"source_document":"FY 2015-16 Budget in Brief","service_area":"Transportation & Parking","bureau":"Portland Bureau of Transportation","key_performance_measures":"BOT – % of bridges in non-distressed condition","fy":"2013-14","budget_type":"Actual ","amount":null,"units":""},
```

Comment:
Could you update this with the following?  I pretty printed it, which isn't totally necessary but if there's a comma after the last item of a collection it isn't valid json.  Python is cool with it but JSON isn't.

```json
[
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Emergency Communications",
    "key_performance_measures": "BOEC - % of priority medical calls dispatched within 90 seconds",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": 72,
    "units": "%"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Fire & Police Disability & Retirement",
    "key_performance_measures": "FPDR - Tax levy rate per $1,000 of Real Market Value",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": 1.62,
    "units": "$"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Transportation & Parking",
    "bureau": "Portland Bureau of Transportation",
    "key_performance_measures": "BOT – % of bridges in non-distressed condition",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": null,
    "units": ""
  }
]
```